infra/terraform/environments/alpha/main.tf

```diff
@@ -86,6 +86,8 @@ module "container_registry" {
 
 module "data_access" {
     source = "../../modules/data_access"
+
+    region = "${var.region}"
     env = "${var.env}"
     sns_arn = "${var.sns_arn}"
     gh_hook_secret = "${var.gh_hook_secret}"
```

Comment:
This secret will have to be populated in the github webhook config, so it should be defined as a Terraform output value so it can be easily extracted.